test/view_component/integration_test.rb

```diff
@@ -38,6 +38,16 @@ def test_rendering_component_in_a_controller
     assert_includes inline_response, baseline_response
   end
 
+  def test_rendering_component_in_a_controller_with_block
+    # Note that in order to do this, you must `tap` the component instance.
+
+    get "/controller_inline_with_block"
```

Comment:
IMO we should remove this comment. If we want to document this approach, we should do so in the component, or in the docs themselves.

```suggestion
    get "/controller_inline_with_block"
    
```